ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-arm.yaml

```diff
@@ -0,0 +1,53 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-aws-413-arm
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  arm64-latest:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  latest:
```

Comment:
@jtaleric Is there any reason to have this latest section here? 2 things I notice:
1. It's not pointing to ARM build
2. It's pointing to 4.14 build and I see this file is specific to 4.13 tests

Reply:
I had just the arm one, but if you remove `latest` you will get an error from the `make update` stating you have to keep `latest`..